.mailmap

```diff
@@ -790,6 +797,7 @@ Nikhil Gopalam <gopalamn@umich.edu> gopalamn <gopalamn@umich.edu>
 Nikhil Maan <nikhilmaan22@gmail.com> Nikhil Maan <nikhilman22@gmail.com>
 Nikhil Pappu <nkhlpappu@gmail.com>
 Nikhil Sarda <diff.operator@gmail.com>
+NikhilSDate <47920034+NikhilSDate@users.noreply.github.com>
```

Comment:
@NikhilSDate what name and email do you want to be recorded in the AUTHORS file?

Reply:
I would want the name to be Nikhil Date and email to be nikhil.s.date@gmail.com.

Reply:
Thanks. I've corrected the .mailmap to show this as an alias for Nikhil Date (which was already listed in the AUTHORS file).

Reply:
Thanks for correcting the .mailmap entry.